app/forms/idv/state_id_form.rb

```diff
@@ -3,9 +3,9 @@ class StateIdForm
     include ActiveModel::Model
     include FormStateIdValidator
 
-    ATTRIBUTES = %i[first_name last_name dob state_id_address1 state_id_address2
-                    state_id_city state_id_zipcode state_id_jurisdiction
-                    state_id_state state_id_number same_address_as_id].freeze
+    ATTRIBUTES = %i[first_name last_name dob identity_doc_address1 identity_doc_address2
+                    identity_doc_city identity_doc_zipcode state_id_jurisdiction
+                    identity_doc_state state_id_number same_address_as_id].freeze
```

Comment:
That's a good point. I guess my hesitation is at the moment we're not quite sure which fields will be generic and state id only. I've heard another form of ID might be the passport. I'm not sure what forms of ID will be allowed beyond that, and I'm unsure which fields would be used in multiple IDs.

That leads me to believe maybe that comment should wait until we have another ID in place to see what's shared or unique, but I'm curious what you think. As I haven't worked with the code much yet, you might have a good idea of if that comment would be helpful sooner rather than later.